src/app/actions.ts

```diff
@@ -2,7 +2,7 @@
 
 import speciesData from "@/species.json";
 import flattenedRegionCodes from "@/flattenedRegionCodes.json";
-import Fuse, { FuseResult } from "fuse.js";
+import Fuse from "fuse.js";
```

Comment:
Turns out this import was unused! Nice lil tidy up imo.

Reply:
broken window fix 😄